build.go

```diff
@@ -131,16 +133,32 @@ func main() {
 	}
 }
 
+func packageGrafana() {
+	platformArg := fmt.Sprintf("--platform=%v", goos)
+	previousPkgArch := pkgArch
+	if pkgArch == "" {
+		pkgArch = goarch
+	}
+	postProcessArgs := gruntBuildArg("package")
+	postProcessArgs = append(postProcessArgs, platformArg)
+	grunt(postProcessArgs...)
+	pkgArch = previousPkgArch
+
+	if goos == "linux" && goarch == "amd64"{
```

Comment:
Is this a temporary workaround to avoid arm deb package with armv7 arch instead of armhf ?
If yes, nothing to say about that but if this means you intend to publish only tarball for armv7, I have to complain. deb (and rpm) packages are much more convenient.

Reply:
Yeah its just a workaround until I or someone else has had the chance to look into what we need to do to fix that issue. My hope is to be able to merge this PR today so that we can start getting feedback from our users.